frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1776,6 +1785,9 @@ function ReaderFooter:onExitFlippingMode()
 end
 
 function ReaderFooter:onTapFooter(ges)
+    if self.settings.lock_tap then
+        return true
```

Comment:
Returning `true` will prevent the tap from being handled by other code. After it, there are page turns and bottom menu (see #5789).
So, may be just `return` as with next `if self.has_no_mode `.

Reply:
You're right :+1: